static/third/bootstrap-typeahead/typeahead.js

```diff
@@ -307,17 +307,17 @@
 
   , listen: function () {
       this.$element
-        .on('blur',     $.proxy(this.blur, this))
-        .on('keypress', $.proxy(this.keypress, this))
-        .on('keyup',    $.proxy(this.keyup, this))
+        .on('blur',     this.blur(this))
+        .on('keypress', this.keypress.bind(this))
+        .on('keyup',    this.keyup.bind(this))
 
       if (this.eventSupported('keydown')) {
-        this.$element.on('keydown', $.proxy(this.keydown, this))
+        this.$element.on('keydown', this.keydown.bind(this))
       }
 
       this.$menu
-        .on('click', $.proxy(this.click, this))
-        .on('mouseenter', 'li', $.proxy(this.mouseenter, this))
+        .on('click', this.click.bind(this))
+        .on('mouseenter', 'li', this.mouseenter.bind(this))
```

Comment:
I guess the theory is that we're maintaining this typeahead code at this point?  OK.